charts/akeyless-api-gateway/templates/deployment.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: {{ .Values.kind }}
```

Comment:
why? this template is "deployment" kind by definition

Reply:
this customer wants to run the gw/injector as DeamonSet and not Deployment
https://akeyless.atlassian.net/browse/ASM-6249

Reply:
OK, so it is better to create a flag on values.yaml file and make a different yaml for Daemonset so it will be easier to maintain.